projects/vdk-plugins/vdk-jobs-troubleshooting/src/vdk/plugin/jobs_troubleshoot/troubleshoot_utilities/healthcheck_server.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from http.server import HTTPServer
+from http.server import SimpleHTTPRequestHandler
+from threading import Thread
+from typing import Any
+
+
+class HealthCheckServer:
```

Comment:
Provide descripton of the class purpose, usage, examples

Reply:
Done.